BrainPortal/public/javascripts/userfiles.js

```diff
@@ -310,31 +252,31 @@ $(function() {
     update: function (form) {
       var uform = userfiles.children('form');
 
-      $(form).find('select').each(function () {
-        var select = $(this);
-
-        /*
-         * disabled select boxes correspond to an explicitly empty field; send
-         * an empty value.
-         */
-        if (select.prop('disabled')) {
-          $(form).append(
-            $('<input type="hidden" value="" />')
-              .attr('name', select.attr('name'))
-          );
+          $(form).find('select').each(function () {
```

Comment:
Quick note, these changes right here are cause by blank space, use `?w=1` to preview the diff properly. I suspect tabs were the cause.